### DISCUSSION TITLE
Holding back major upgrades to avoid applications being disabled

### DISCUSSION THREAD

**Discussion by mikebakke** [nextcloud/all-in-one#1718] [Ideas] [answered]
I recently stopped my master to perform maintenance and on restart was updated to v25 with no option to decline.

The issue is that after restart I found at least 5 applications which the system disabled as not being compatible. This is not a good approach as it removes control from the user over the system. I was keen to get up to 25 but did not gain enough in features to outweigh the losses. No doubt the apps will gradually get fixed but until them I am stuck.

Please - consider an approach where before an update is triggered, the user is informed which apps will be disabled and simply offer the choice to "freeze" the system in it's current state until released by the user.

Until then, my only thought is to put a specific tag in compose instead of :latest and bump it manually after changelog reviews. Inconvenient but far preferable to losing (even temporarily) a key application.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://help.nextcloud.com/t/nextcloud-aio-possible-to-hold-back-major-updates/148115/9
